sdk/schemaregistry/azure-schemaregistry-avroserializer/azure/schemaregistry/serializer/avroserializer/_schema_registry_avro_serializer.py

```diff
@@ -155,21 +156,21 @@ def serialize(self, data, schema, **kwargs):
         stream.close()
         return payload
 
-    def deserialize(self, data, **kwargs):
+    def deserialize(self, value, **kwargs):
         # type: (bytes, Any) -> Dict[str, Any]
         """
         Decode bytes data.
 
-        :param bytes data: The bytes data needs to be decoded.
+        :param bytes value: The bytes data needs to be decoded.
         :rtype: Dict[str, Any]
```

Comment:
as per OOB discussion, we need to investigate supporting input of Any type (python primitive types) first, which could change this API to return Any type